code/modules/mob/living/carbon/human/intoxication.dm

```diff
@@ -0,0 +1,42 @@
+#define		AE_DIZZY 		5
+#define 	AE_SLURRING 	10
+#define 	AE_CONFUSION 	18
+#define 	AE_BLURRING 	30
+#define 	AE_DROWSY 		55
```

Comment:
Can you add a define for like AE_VOMIT which gives you a chance of vomiting at around 55-80?